index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <link href="css/style.css" rel="stylesheet" type="text/css">
+    <meta charset="utf-8">
+    <title></title>
+  </head>
+  <body>
+    <div class="navbar">
+      <img src="http://placehold.it/200x70"><h1>BusMall Survey</h1>
```

Comment:
These two tags should be on separate lines.  In HTML, generally only li's and a's will be on the same line as whatever they are holding / within (usually an h tag)